Gemfile

```diff
@@ -73,7 +72,7 @@ gem 'bootstrap-sass', '~> 3.3.5'
 
 gem 'blacklight', '~> 6.3'
 gem 'blacklight-gallery', '~> 0.3'
-gem 'blacklight-maps', '0.2.0'
+gem 'blacklight-maps', '0.5.0'
```

Comment:
I don't think we should upgrade blacklight-maps yet. I don't think we have indexed the data in a way that'll work with the new geojson-based whatnot.